label_studio/tasks/api.py

```diff
@@ -298,23 +298,29 @@ class AnnotationDraftAPI(RequestDebugLogMixin, generics.RetrieveUpdateDestroyAPI
 
 
 @method_decorator(name='list', decorator=swagger_auto_schema(
-    tags=['Predictions'], operation_summary="List predictions",
+    tags=['Predictions'],
+    operation_summary="List predictions",
     operation_description="List all predictions."))
 @method_decorator(name='create', decorator=swagger_auto_schema(
-    tags=['Predictions'], operation_summary="Create prediction",
+    tags=['Predictions'],
+    operation_summary="Create prediction",
     operation_description="Create a prediction."))
 @method_decorator(name='retrieve', decorator=swagger_auto_schema(
-    tags=['Predictions'], operation_summary="Get prediction",
-    operation_description="Get all predictions."))
+    tags=['Predictions'],
+    operation_summary="Get prediction",
+    operation_description="Get all predictions for a specific task ID."))
```

Comment:
@triklozoid can you clarify -- does this use the task ID, project ID, ML backend ID, or something else? It's unclear to me from the code for the serializer.

Reply:
Good question, now you can get only all predictions from organization. May be it's useful, I'll add filters by task and project.

https://github.com/heartexlabs/label-studio/pull/1130

How it works after this PR: 
```
GET /api/predictions  - all predictions from organization
GET /api/predictions?task=TASK_ID - all predictions for task with id TASK_ID
GET /api/predictions?task__project=PROJECT_ID - all predictions for project with id PROJECT_ID
```

Reply:
Thank you!! I'll update accordingly :)